static/bidder-info/richaudience.yaml

```diff
@@ -0,0 +1,11 @@
+maintainer:
+  email: partnerintegrations@richaudience.com
+gvlVendorID: 108
+modifyingVastXmlAllowed: true
+capabilities:
+  site:
+    mediaTypes:
+      - banner
+userSync:
+  iframe:
+    url: "https://sync.richaudience.com/74889303289e27f327ad0c6de7be7264/?consentString={{.GDPRConsent}}&r={{.RedirectURL}}[PDID]"
```

Comment:
I'd just like to confirm that `74889303289e27f327ad0c6de7be7264` identifies the Prebid Ecosystem in general and not a specific host, such as Xandr/AppNexus.